cityofaustin/atd-moped
# Re-implement project summary map

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/10995
https://github.com/cityofaustin/atd-data-tech/issues/10689

This PR updates the project summary map to show features that come from the `project_geography` view. I also updated the map to use some of the same config and patterns as the components map. I removed the `ErrorBoundary` component from the summary view since it isn't a pattern that we've used elsewhere in the app. Definitely up for debate because having those can be handy, but I think that if we remove it then I should remove the package for it as well.

## Testing
**URL to test:** 

local or https://md-10995-summary-map--atd-moped-main.netlify.app/moped/

**Steps to test:**
1. Create a new project and, when you reach the summary page, notice that the fallback component with the map and scooter are showing instead of the summary map'
2. Click the "Add Components" button in the fallback and use the map to add some components with features of point and line type
3. Go back to the summary page, and you should see all of the component features that you added and the map should zoom to them just like in the components map
4. Try out the speed dial selector in the bottom right of the summary map and switch between street and aerial

---
#### Ship list
